akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FormFieldDirectivesSpec.scala

```diff
@@ -12,6 +12,9 @@ import akka.http.scaladsl.model._
 import MediaTypes._
 
 class FormFieldDirectivesSpec extends RoutingSpec {
+  // FIXME: unfortunately, it has make a come back, this time it's reproducible ...
+  import akka.http.scaladsl.server.directives.FormFieldDirectives.FieldMagnet
```

Comment:
It's the old problem where a compilation error happens depending on the order in which files are compiled. I thought I fixed it in #17552 but now it came back and I haven't yet found the reason.

Here is some background: https://groups.google.com/d/topic/scala-language/KZhcHb6e8DI/discussion 

Yes, I'll create a ticket.